src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    private Person person;
+    private Tag tag;
+    private String status;
+    
+    private static final String TAG_ADD = "+";
+    private static final String TAG_REMOVE = "-";
+    
+    public Tagging(Person person, Tag tag, String status) {
+    	this.person = person;
+    	this.tag = tag;
+    	this.status = status;
+    }
+    
+    public String toString() {
+    	StringBuilder tagging = new StringBuilder();
+    	if(status == "ADD") {
+    		tagging.append(TAG_ADD);
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.